acis_thermal_check/tests/test_dea.py

```diff
@@ -0,0 +1,18 @@
+import os
+import shutil
+import tempfile
+
+from acis_thermal_check import dea_check
```

Comment:
Astropy uses relative imports everywhere including tests (despite specific advice to the contrary in the pytest docs).  My own experience is that pytest never fails to surprise me with unexpected behavior.  So I would do `from .. import dea_check` and then there is absolutely no question about getting the local module and not the system-installed one.  Likewise below `from .utils import ...`.

If you more confident than me then an absolute import is fine.  :smile: